bug:make install does not exit without requirement tools

When compiling and install kcp without jq tool on RHEL 8.6 VM,  it doesn't exit. Proceed to complete installation, executing kcp runs into panic

Version:
kcp version v1.24.3+kcp-v0.10.0

How Reproducible:
Always

Steps to Reproduce:
1.clone the kcp repo
#git clone https://github.com/kcp-dev/kcp.git
Checkout v0.10.0
#cd kcp 
#git checkout -b  v0.10.0 v0.10.0

2.Compiling the kcp
#make install
bash: jq: command not found
bash: jq: command not found
bash: jq: command not found
GOOS=linux GOARCH=amd64 go install -ldflags="-X k8s.io/client-go/pkg/version.gitCommit=25254541 -X k8s.io/client-go/pkg/version.gitTreeState=clean -X 
…
go: downloading github.com/kcp-dev/kubernetes/staging/src/k8s.io/mount-utils v0.0.0-20221208163042-f7f1c9288c3b
ln -sf /root/go/bin/kubectl-workspace /root/go/bin/kubectl-ws
ln -sf /root/go/bin/kubectl-workspace /root/go/bin/kubectl-workspaces

#kcp
panic: version string "+kcp-v0.10.0" doesn't match expected regular expression: "^v(\d+\.\d+\.\d+)"

goroutine 1 [running]:
k8s.io/component-base/metrics.parseVersion({{0x39c0a6e, 0x0}, {0x39c0a6f, 0x0}, {0x39c13a0, 0xc}, {0x39c1380, 0x8}, {0x39c0ae4, 0x5}, ...})
	/root/go/pkg/mod/github.com/kcp-dev/kubernetes/staging/src/k8s.io/component-base@v0.0.0-20221208163042-f7f1c9288c3b/metrics/version_parser.go:47 +0x274
k8s.io/component-base/metrics.newKubeRegistry({{0x39c0a6e, 0x0}, {0x39c0a6f, 0x0}, {0x39c13a0, 0xc}, {0x39c1380, 0x8}, {0x39c0ae4, 0x5}, ...})
	/root/go/pkg/mod/github.com/kcp-dev/kubernetes/staging/src/k8s.io/component-base@v0.0.0-20221208163042-f7f1c9288c3b/metrics/registry.go:320 +0x119
k8s.io/component-base/metrics.NewKubeRegistry()
	/root/go/pkg/mod/github.com/kcp-dev/kubernetes/staging/src/k8s.io/component-base@v0.0.0-20221208163042-f7f1c9288c3b/metrics/registry.go:335 +0x78
k8s.io/component-base/metrics/legacyregistry.init()
	/root/go/pkg/mod/github.com/kcp-dev/kubernetes/staging/src/k8s.io/component-base@v0.0.0-20221208163042-f7f1c9288c3b/metrics/legacyregistry/registry.go:29 +0x1d

Expected Results:
Without jq, we should exit the compiling kcp 

Additional Info:
After the jq tool was installed, retried, the issue was gone.

--- a/Makefile
+++ b/Makefile
@@ -100,7 +100,7 @@ ldflags:
 
 .PHONY: require-%
 require-%:
-	@if ! command -v $* 1> /dev/null 2>&1; then echo "$* not found in \$$PATH"; exit 1; fi
+	@if ! command -v $* 1> /dev/null 2>&1; then echo "$* not found in ${PATH}"; exit 1; fi
 
 build: WHAT ?= ./cmd/... ./tmc/cmd/...
 build: require-jq require-go require-git verify-go-versions ## Build the project
@@ -122,7 +122,7 @@ build-kind-images: build-kind-images-ko
 	@test -n "$(TEST_IMAGE)" && (echo $(TEST_IMAGE) pushed to "$(KIND_CLUSTER_NAME)" kind cluster) || (echo Failed to create test image and and/or to push it to "$(KIND_CLUSTER_NAME)" kind cluster; exit 1)
 
 install: WHAT ?= ./cmd/...
-install:
+install: require-jq require-go require-git verify-go-versions ## Install the project
 	GOOS=$(OS) GOARCH=$(ARCH) CGO_ENABLED=0 go install -ldflags="$(LDFLAGS)" $(WHAT)
 	ln -sf $(INSTALL_GOBIN)/kubectl-workspace $(INSTALL_GOBIN)/kubectl-ws
 	ln -sf $(INSTALL_GOBIN)/kubectl-workspace $(INSTALL_GOBIN)/kubectl-workspaces

